### ISSUE TITLE
CI seems broken

### ISSUE THREAD

**Issue by videlec** [flatsurf/surface-dynamics#98]
The CI fails while installing conda with
```
Warning: Future Miniforge releases will NOT build Mambaforge installers. We advise you switch to Miniforge at your earliest convenience. More details at [https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/.](https://conda-forge.org/news/2024/07/29/sunsetting-mambaforge/)
  Warning: ERROR: executing pre_install.sh failed
  
  ERROR: executing pre_install.sh failed
Error: The process '/usr/bin/bash' failed with exit code 1
```
See eg https://github.com/flatsurf/surface-dynamics/actions/runs/11120372070/job/30915370755

**Comment by videlec** [comment]
@saraedum any simple work around?

**Comment by saraedum** [comment]
I think you can just remove the `miniforge-variant` from all workflows.

**Comment by videlec** [comment]
Thanks! Fixed in [edccf62](https://github.com/flatsurf/surface-dynamics/pull/97/commits/edccf62480f27c8fac03c8a839e8898f982c2245) from #97.
